src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -59,6 +60,22 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
     setDeepValue(filterRefs.current, filterName, ref);
   }
 
+  function GetActions(position: ActionsPosition) {
```

Comment:
Please create a normal functional component instead of a function returning JSX.

Reply:
@neotrow Done, ti looks better to you now?